lib/util/pssh.js

```diff
@@ -139,11 +155,23 @@ shaka.util.Pssh = class {
     psshBox.set(data, byteCursor);
     byteCursor += dataLength;
 
-    goog.asserts.assert(byteCursor === psshSize, 'MS PRO invalid length.');
+    goog.asserts.assert(byteCursor === psshSize, 'PSSH invalid length.');
     return psshBox;
   }
 
 
+  /**
+   * Check if inidata contains Pssh
+   * @param {!Uint8Array} initData
+   * @param {boolean=} extractMoov boolean to extract pssh from moov atom
```

Comment:
I think you can drop extractMoov here, too.

Reply:
Yes, I'm going to remove it.